web/src/elements/forms/SearchSelect/ak-search-select.ts

```diff
@@ -150,6 +156,7 @@ export class SearchSelect<T> extends CustomEmitterElement(AKElement) {
                 objects.forEach((obj) => {
                     if (this.selected && this.selected(obj, objects || [])) {
                         this.selectedObject = obj;
+                        this.dispatchCustomEvent("ak-change", { value: this.selectedObject });
```

Comment:
This was the bug Marc found: we weren't propagating the event "This object's value changed from *undefined* to `selectedObject`" on start-up, so it wasn't reaching `ak-form-element-horizontal` if it wasn't changed.